src/main/java/com/mateacademy/factory_method/BuilderInterface.java

```diff
@@ -0,0 +1,13 @@
+package com.mateacademy.factory_method;
+
+public interface BuilderInterface {
```

Comment:
do you really neeed this interface&

Reply:
my static nested class in BuilderProduct class implements this interface, so if we want to create one more class like BuilderProduct we just should implement this interface